kubernetes/deployment-production-azure-canary.tmpl

```diff
@@ -0,0 +1,154 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: panoptes-production-azure-canary-app
+  labels:
+    app: panoptes-production-azure-canary-app
+spec:
+  selector:
+    matchLabels:
+      app: panoptes-production-azure-canary-app
+  template:
+    metadata:
+      labels:
+        app: panoptes-production-azure-canary-app
+    spec:
+      containers:
+        - name: panoptes-production-azure-canary-app
+          image: zooniverse/panoptes:__IMAGE_TAG__
+          resources:
+            requests:
+              memory: "700Mi"
+              cpu: "50m"
+            limits:
+              memory: "700Mi"
+              cpu: "1000m"
```

Comment:
If this is just for internal team testing the i think we can get away with half a vcpu. If response times are impacted it's not user facing. 
```suggestion
              cpu: "500m"
```

Reply:
Makes sense 👍